<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>progressTestClass</class>
 <widget class="QWidget" name="progressTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>630</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>progressTest</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#progressTestClass{
	background-color: rgb(0, 0, 0);
}

QPushButton
{
background: #01EEC3;
border-radius: 16px;
font-size: 12px;
font-family: 微软雅黑;
color: #212426;
}
</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>550</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="vProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>580</width>
     <height>32</height>
    </rect>
   </property>
   <property name="value">
    <number>90</number>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>660</y>
     <width>560</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="MyProcessBar" name="progressBar_new">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>580</width>
     <height>32</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QRoundProgressBar" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>230</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="AnimationProcessBar" name="progressBar_animation">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>580</width>
     <height>32</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="vProgressBars" name="progressBar_radius">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>580</width>
     <height>32</height>
    </rect>
   </property>
   <property name="value">
    <number>90</number>
   </property>
  </widget>
  <widget class="vWaterProgressBar" name="widget_water" native="true">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="vWaterProgressBar" name="widget_water_2" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="BatteryProgressBar" name="widget_battery" native="true">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>430</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="vpkProgressBar" name="progressBar_pk">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>600</y>
     <width>580</width>
     <height>32</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="RoundProgressBar" name="widget_R" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>vProgressBar</class>
   <extends>QProgressBar</extends>
   <header location="global">vProgressBar.h</header>
  </customwidget>
  <customwidget>
   <class>MyProcessBar</class>
   <extends>QProgressBar</extends>
   <header location="global">myprocessbar.h</header>
  </customwidget>
  <customwidget>
   <class>QRoundProgressBar</class>
   <extends>QWidget</extends>
   <header location="global">qroundprogressbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AnimationProcessBar</class>
   <extends>QProgressBar</extends>
   <header location="global">animationprocessbar.h</header>
  </customwidget>
  <customwidget>
   <class>vProgressBars</class>
   <extends>QProgressBar</extends>
   <header location="global">vProgressBars.h</header>
  </customwidget>
  <customwidget>
   <class>vWaterProgressBar</class>
   <extends>QWidget</extends>
   <header location="global">vwaterprogressbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BatteryProgressBar</class>
   <extends>QWidget</extends>
   <header location="global">batteryprogressbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>vpkProgressBar</class>
   <extends>QProgressBar</extends>
   <header location="global">vpkprogressbar.h</header>
  </customwidget>
  <customwidget>
   <class>RoundProgressBar</class>
   <extends>QWidget</extends>
   <header location="global">roundprogressbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="progressTest.qrc"/>
 </resources>
 <connections/>
</ui>
